bench/irmin-pack/tree.ml

```diff
@@ -238,7 +412,24 @@ struct
       Trees_trace.add_commits repo commits |> Benchmark.run config
     in
     let+ () = Store.Repo.close repo in
-    (config, result)
+    prepare_results_dir config.results_dir;
+    let json_channel =
+      let ( / ) = Filename.concat in
+      config.results_dir / "boostrap_trace.json" |> open_out
+    in
+    Format.fprintf
+      (Format.formatter_of_out_channel json_channel)
+      "%a" Trees_trace.pp_stats
+      (true, config.flatten, config.inode_config);
```

Comment:
```suggestion
    Format.fprintf
      (Format.formatter_of_out_channel json_channel)
      "%a%!" Trees_trace.pp_stats
      (true, config.flatten, config.inode_config);
```

If you create a formatter this way, you need to flush to the underlying channel before closing it.

Reply:
🙈 The truth is that I have no knowledge of `Format` and `Fmt`. I have to read up on the subject. I push this to the top of my TODO. Thank you very much for the review!